tests/Builder/SourceFileRepositoryTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\Builder;
+
+use Doctrine\Website\Builder\SourceFile;
+use Doctrine\Website\Builder\SourceFileRepository;
+use PHPUnit\Framework\TestCase;
+use function usort;
+
+class SourceFileRepositoryTest extends TestCase
+{
+    /** @var string */
+    private $rootDir;
+
+    /** @var SourceFileRepository */
+    private $sourceFileRepository;
+
+    public function testGetFiles() : void
+    {
+        $buildDir = '';
+        $inPath   = 'source/blog';
+
+        $files = $this->sourceFileRepository->getFiles(
+            $buildDir,
+            $inPath
```

Comment:
`$inPath` is optional and we don't have a test that covers omitted second argument.